ground/gcs/src/plugins/boards_dtf/dtfc.h

```diff
@@ -36,37 +36,35 @@
 
 #include <coreplugin/iboardtype.h>
 
-class IBoardType;
-
 class Dtfc : public Core::IBoardType
 {
 public:
     Dtfc();
     virtual ~Dtfc();
 
-    virtual QString shortName();
-    virtual QString boardDescription();
-    virtual bool queryCapabilities(BoardCapabilities capability);
-    virtual QPixmap getBoardPicture();
-    virtual QString getHwUAVO();
-    virtual int queryMaxGyroRate();
+    QString shortName();
+    QString boardDescription();
```

Comment:
Why is the 'virtual' keyword removed in these types of places?  We don't do it everywhere, but I like it because of http://stackoverflow.com/questions/1370976/c-style-prefixing-virtual-keyword-to-overridden-methods   ... in any case this does not seem to be minimal/required